puppetboard/app.py

```diff
@@ -549,7 +551,9 @@ def report_latest(env, node_name):
     check_env(env, envs)
 
     if env == '*':
-        node_query = '["=", "certname", "{0}"]'.format(node_name)
+        query='["and",' \
+            '["=", "certname", "{0}"],' \
+            '["=", "latest_report?", true]]'.format(node_name)
```

Comment:
We cannot accept this PR with the current changes to the ``report_latest()`` function. Please keep it just to the Overview